app/jobs/in_person/send_proofing_notification_job.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+module InPerson
+  class SendProofingNotificationJob < ApplicationJob
+    # @param [String] enrollment_id primary key of the enrollment
+    def perform(enrollment_id)
+      return true if IdentityConfig.store.in_person_proofing_enabled.blank? ||
+                     IdentityConfig.store.in_person_send_proofing_notifications_enabled.blank?
+      enrollment = InPersonEnrollment.find(
+        enrollment_id,
+        include: [:notification_phone_configuration, :user],
+      )
+      # skip
+      if !enrollment.notification_phone_configuration.present? ||
+         (!enrollment.passed? && !enrollment.failed?)
+        # log event
+        analytics(user: enrollment.user).
+          idv_in_person_usps_proofing_results_notification_job_skipped(
+            enrollment_code: enrollment.enrollment_code,
+            enrollment_id: enrollment.id,
+          )
+        return true
+      end
+      analytics(user: enrollment.user).
+        idv_in_person_usps_proofing_results_notification_job_started(
+          enrollment_code: enrollment.enrollment_code,
+          enrollment_id: enrollment.id,
+        )
+      if enrollment.passed? || enrollment.failed?
+        # send notification and log result
+        phone = enrollment.notification_phone_configuration.formatted_phone
+        message = notification_message(enrollment: enrollment)
+        response = Telephony.send_notification(
+          to: phone, message: message,
+          country_code: Phonelib.parse(phone).country
+        )
+        handle_telephony_result(enrollment: enrollment, phone: phone, telephony_result: response)
+        if response.success?
+          # if notification sent successful
+          enrollment.update(
+            notification_sent_at: Time.zone.now,
+          )
+          # delete notification phone configuraiton if success
+          enrollment.notification_phone_configuration.destroy!
+        end
+      end
+      return true
```

Comment:
I think we should remove this, return values not used in async jobs
```suggestion
```

Reply:
@zachmargolis fixed.